boards.txt

```diff
@@ -442,6 +442,12 @@ BLENano.menu.softdevice.s130.upload.maximum_size=151552
 BLENano.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 BLENano.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_{build.chip}.ld
 
+BLENano.menu.lfclk.lfxo=Crystal Oscillator
```

Comment:
I think it's simpler to add -DUSE_LFXO to extra_flags because the Crystal Oscillator is hard wired.